src/activities/Elsa.Activities.SQL/Activities/SqlCommand/ExecuteSqlServerCommand.cs

```diff
@@ -0,0 +1,60 @@
+using Elsa.Services;
+using Elsa.Attributes;
+using Elsa.Services.Models;
+using Elsa.ActivityResults;
+using Elsa.Design;
+using Elsa.Expressions;
+using Elsa.Activities.Sql.Factory;
+
+namespace Elsa.Activities.Sql.Activities
+{
+    /// <summary>
+    /// Execute an SQL query on given database using connection string
+    /// </summary>
+    [Trigger(
+        Category = "SQL Server",
+        DisplayName = "Execute Sql Command",
+        Description = "Execute given SQL command and returned number of rows affected",
+        Outcomes = new string[0]
+    )]
+    public class ExecuteSqlServerCommand : Activity
+    {
+        /// <summary>
+        /// SQl script to execute
+        /// </summary>
+        [ActivityInput(
+            Hint = "SQL command to execute",
+            UIHint = ActivityInputUIHints.MultiLine,
+            SupportedSyntaxes = new[] { SyntaxNames.JavaScript, SyntaxNames.Liquid }
+        )]
+        public string Command { get; set; } = default!;
+
+        /// <summary>
+        /// Connection string to run SQL
```

Comment:
Here we could include a note that one needs to be aware of the breaking SqlClient v4 change.